src/box/lua/schema.lua

```diff
@@ -3291,7 +3316,9 @@ box.internal.collation.create = function(name, coll_type, locale, opts)
     end
     local lua_opts = {if_not_exists = opts.if_not_exists }
     check_param_table(lua_opts, {if_not_exists = 'boolean'})
+    check_schema_version(opts)
```

Comment:
Removed `box.internal.collation` from the patch.

We discussed the list of functions in person and decided that privilege manipulation functions should not be forbidden, but I can't remember why exactly. @sergepetrenko, can you advise?

Reply:
It was @alyapunov's point. Loosely speaking "the user might want to set up replication after upgrading tarantool but before issuing `box.schema.upgrade()` and that might require granting privileges in some rare cases".

Although this could be dont via a plain `box.space._priv:replace` as well.